frontend/public/locales/en/translation.json

```
@@ -1223,6 +1223,7 @@
   "Enable policy": "Enable policy",
   "Enable search to display application statuses properly.": "Enable search to display application statuses properly.",
   "Enable this option to instruct Submariner not to access any external servers for public IP resolution.": "Enable this option to instruct Submariner not to access any external servers for public IP resolution.",
+  "Enable this option to use Custom Submariner Subscription.": "Enable this option to use Custom Submariner Subscription.",
```

```suggestion
  "Enable this option to use a custom Submariner subscription.": "Enable this option to use a custom Submariner subscription.",
```
Unless "Custom Submariner Subscription" is the name of a product or feature that is capitalized like this elsewhere, I would use lowercase mostly.

---

Changed to lower case.